### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by maksimpro001**

### CODE REVIEW

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]
The solution won't work because at the moment it will only copy files with the same name, and should only copy files with different names, as the task description says. Fix that, please

File: app/main.py [hunk 1]
```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
+    if text[0] != "cp":
+        return
+    if text[1] == text[2]:
+        return
```
Comment (NikOneZ1) [inline app/main.py:6]: You able to merge this two conditions
```suggestion
    if text[0] != "cp" or text[1] == text[2]:
        return
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,8 @@
...
+    with open(text[1], "r") as f1, open(text[2], "w") as f2:
+        f2.write(f1.read())
```
Comment (NikOneZ1) [inline app/main.py:8]: Use more convenient naming 
```suggestion
    with open(text[1], "r") as file1, open(text[2], "w") as file2:
        file2.write(file1.read())
```

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
```
Comment (EdAlekseiev) [inline app/main.py:2]: `" "` is default value for `split`

Comment (EdAlekseiev) [inline app/main.py:2]:
```suggestion
    text = command.split()
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by NikOneZ1 — APPROVED** [review]
Well done!

**Pull request closed without merging**